packages/neuron-ui/package.json

```diff
@@ -48,6 +48,7 @@
     "i18next": "21.10.0",
     "immer": "9.0.21",
     "jsqr": "1.4.0",
+    "@nervosnetwork/neuron-shared": "0.110.1",
```

Comment:
Github suggestion is not available here, so it should be updated manually.

This dependency should be placed beneath `"@nervosnetwork/ckb-sdk-utils": "0.107.0",` because dependencies imported by `lerna` are ordered alphanumerically.